packages/hoprd/src/api/v3/paths/healthyz/index.ts

```diff
@@ -0,0 +1,60 @@
+import type { Hopr } from '@hoprnet/hopr-utils'
+import type { Operation } from 'express-openapi'
+import { STATUS_CODES } from '../../utils.js'
+
+export const GET: Operation = [
+  async (req, res, _next) => {
+    const { node }: { node: Hopr } = req.context
+
+    try {
+      if (node.isRunning()) {
```

Comment:
All three endpoints are using `node.isRunning()`, is that intended?

Reply:
Is there a ticket tracking this missing distinction?

Reply:
It's not a feature per se. Should be part of the discussions for 2024.

Reply:
https://github.com/hoprnet/hoprnet/issues/5807